README.md

```diff
@@ -3,37 +3,37 @@ Fill in the text below with your own answers to each question. Add your answers
 
 ## About You
 What are your names, as they appear on the class roster?
-> ...
+> Thu Nguyen
 
 What do you prefer to be called?
-> ...
+> Thu
 
 Why are you taking this course?
-> ...
+> intended major
 
 What are your concerns about this course?
-> ...
+> workload
```

Comment:
I hope the workload will not be too high for this class; I do expect there to be a lot of programming so you can learn the material by actually using it, but I won't be giving you much other work for this course.